gui/mapviewer.py

```diff
@@ -176,7 +176,7 @@ def on_draw(self):
         # draw noises
         for noise in self.world.noises + self.world.old_noises:
             radius = max(0, noise.radius - (self.world.time_ticks - noise.time) * self.world.TIME_PER_TICK * 0.5 * noise.radius)
-            if radius > 0:# and not noise.drawn:
+            if radius > 0:  # and not noise.drawn:
```

Comment:
Looks like my pep8 checker in my IDE isn't perfect :p

Reply:
i can't help it, especially when it's comments :D